src/setOfPeople.js

```diff
@@ -39,6 +39,11 @@
  */
 function setOfPeople(people) {
   // write code here
+  const inRoom = new Set();
+  people.forEach(person =>
+    (!inRoom.has(person)) ? inRoom.add(person) : inRoom.delete(person)
```

Comment:
it is a bad way to wrap line and brackets still unnecessary
```javascript
  people.forEach(person => (
    !inRoom.has(person) ? inRoom.add(person) : inRoom.delete(person)
  ));
```
Or
```javascript
  people.forEach(person => (
    !inRoom.has(person)
      ? inRoom.add(person)
      : inRoom.delete(person)
  ));
```

Reply:
Ok)!